src/helpers/parseTranscriptionData.js

```diff
@@ -32,28 +34,50 @@ function mapExtractsToReductions(
   reductionIndex = 0,
   reductionText = [],
   subjectIndex = 0,
+  extractUsers = {}
 ) {
   const result = []
+  reduction.user_ids && reduction.user_ids.forEach((userId, idIndex) => {
+    // lets find that user's annotation
+    const userClassificationsOnSubject = extractsByUser[userId]
+    for (let i = 0; i < userClassificationsOnSubject.length; i++) {
```

Comment:
Getting TypeError warning `Cannot read property 'length' of undefined at parseTranscriptionData.js:43` I think related to if this is undefined.